cluster-autoscaler/simulator/clustersnapshot/clustersnapshot_test.go

```diff
@@ -695,3 +748,34 @@ func TestPVCClearAndFork(t *testing.T) {
 		})
 	}
 }
+
+func TestWithForkedSnapshot(t *testing.T) {
+	testCases := validTestCases(t)
+	for name, snapshotFactory := range snapshots {
+		for _, tc := range testCases[:1] {
```

Comment:
Let's not depend on filtering out specific test cases like this. Why test just one?

Reply:
Oops, sorry it's a leftover from debugging, thanks for catching it!